apps/ui/src/networks/offchain/api/index.ts

```diff
@@ -331,19 +330,26 @@ function formatDelegations(space: ApiSpace): SpaceMetadataDelegation[] {
   );
 
   if (space.delegationPortal) {
+    const network =
+      CHAIN_IDS_TO_NETWORKS[
+        parseInt(space.delegationPortal.delegationNetwork, 10)
+      ] || null;
+
     delegations.push({
-      name: null,
-      apiType:
-        (space.delegationPortal?.delegationType as DelegationType) ?? null,
-      apiUrl: space.delegationPortal?.delegationApi ?? null,
-      contractNetwork: null,
-      contractAddress: space.delegationPortal?.delegationContract ?? null
+      name:
+        space.delegationPortal.delegationType === 'compound-governor'
+          ? 'ERC-20 Votes'
+          : 'Split Delegation',
+      apiType: space.delegationPortal.delegationType,
+      apiUrl: space.delegationPortal.delegationApi,
+      contractNetwork: network,
+      contractAddress: `${network}:${space.delegationPortal.delegationContract}`
```

Comment:
Now we add it twice:
<img width="856" alt="image" src="https://github.com/user-attachments/assets/652aae1d-c9ac-4b09-b309-553b4c01ac74">

I don't think we need to add it here, because `contractNetwork` already handles network, `contractAddress` should be just the address.

Reply:
For onchain spaces we split it and only use address:
https://github.com/snapshot-labs/sx-monorepo/blob/26a108fbc2953ed66fa1c66c8e72ff01631ab808/apps/ui/src/networks/common/graphqlApi/index.ts#L206-L207